docker-compose.yml

```diff
@@ -576,136 +591,23 @@ services:
-      DB_CONNECTION_CONTROL_SCHEMA_TOXO: ${DB_CONNECTION_CONTROL_SCHEMA_TOXO:?}
-      DB_CONNECTION_POOL_SIZE_TOXO: ${DB_CONNECTION_POOL_SIZE_TOXO}
-
-      DB_CONNECTION_ENABLED_TRICH: ${DB_CONNECTION_ENABLED_TRICH}
-      DB_CONNECTION_NAME_TRICH: ${DB_CONNECTION_NAME_TRICH:?}
-      DB_CONNECTION_LDAP_TRICH: ${DB_CONNECTION_LDAP_TRICH:?}
-      DB_CONNECTION_PASS_TRICH: ${DB_CONNECTION_PASS_TRICH:?}
-      DB_CONNECTION_DATA_SCHEMA_TRICH: ${DB_CONNECTION_DATA_SCHEMA_TRICH:?}
-      DB_CONNECTION_CONTROL_SCHEMA_TRICH: ${DB_CONNECTION_CONTROL_SCHEMA_TRICH:?}
-      DB_CONNECTION_POOL_SIZE_TRICH: ${DB_CONNECTION_POOL_SIZE_TRICH}
-
-      DB_CONNECTION_ENABLED_TRITRYP: ${DB_CONNECTION_ENABLED_TRITRYP}
-      DB_CONNECTION_NAME_TRITRYP: ${DB_CONNECTION_NAME_TRITRYP:?}
-      DB_CONNECTION_LDAP_TRITRYP: ${DB_CONNECTION_LDAP_TRITRYP:?}
-      DB_CONNECTION_PASS_TRITRYP: ${DB_CONNECTION_PASS_TRITRYP:?}
-      DB_CONNECTION_DATA_SCHEMA_TRITRYP: ${DB_CONNECTION_DATA_SCHEMA_TRITRYP:?}
-      DB_CONNECTION_CONTROL_SCHEMA_TRITRYP: ${DB_CONNECTION_CONTROL_SCHEMA_TRITRYP:?}
-      DB_CONNECTION_POOL_SIZE_TRITRYP: ${DB_CONNECTION_POOL_SIZE_TRITRYP}
-
-      DB_CONNECTION_ENABLED_VECTOR: ${DB_CONNECTION_ENABLED_VECTOR}
-      DB_CONNECTION_NAME_VECTOR: ${DB_CONNECTION_NAME_VECTOR:?}
-      DB_CONNECTION_LDAP_VECTOR: ${DB_CONNECTION_LDAP_VECTOR:?}
-      DB_CONNECTION_PASS_VECTOR: ${DB_CONNECTION_PASS_VECTOR:?}
-      DB_CONNECTION_DATA_SCHEMA_VECTOR: ${DB_CONNECTION_DATA_SCHEMA_VECTOR:?}
-      DB_CONNECTION_CONTROL_SCHEMA_VECTOR: ${DB_CONNECTION_CONTROL_SCHEMA_VECTOR:?}
-      DB_CONNECTION_POOL_SIZE_VECTOR: ${DB_CONNECTION_POOL_SIZE_VECTOR}
-
-      DB_CONNECTION_ENABLED_ORTHO: ${DB_CONNECTION_ENABLED_ORTHO}
-      DB_CONNECTION_NAME_ORTHO: ${DB_CONNECTION_NAME_ORTHO:?}
-      DB_CONNECTION_LDAP_ORTHO: ${DB_CONNECTION_LDAP_ORTHO:?}
-      DB_CONNECTION_PASS_ORTHO: ${DB_CONNECTION_PASS_ORTHO:?}
-      DB_CONNECTION_DATA_SCHEMA_ORTHO: ${DB_CONNECTION_DATA_SCHEMA_ORTHO:?}
-      DB_CONNECTION_CONTROL_SCHEMA_ORTHO: ${DB_CONNECTION_CONTROL_SCHEMA_ORTHO:?}
-      DB_CONNECTION_POOL_SIZE_ORTHO: ${DB_CONNECTION_POOL_SIZE_ORTHO}
-
-      DB_CONNECTION_ENABLED_CLINEPI: ${DB_CONNECTION_ENABLED_CLINEPI}
-      DB_CONNECTION_NAME_CLINEPI: ${DB_CONNECTION_NAME_CLINEPI:?}
-      DB_CONNECTION_LDAP_CLINEPI: ${DB_CONNECTION_LDAP_CLINEPI:?}
-      DB_CONNECTION_PASS_CLINEPI: ${DB_CONNECTION_PASS_CLINEPI:?}
-      DB_CONNECTION_DATA_SCHEMA_CLINEPI: ${DB_CONNECTION_DATA_SCHEMA_CLINEPI:?}
-      DB_CONNECTION_CONTROL_SCHEMA_CLINEPI: ${DB_CONNECTION_CONTROL_SCHEMA_CLINEPI:?}
-      DB_CONNECTION_POOL_SIZE_CLINEPI: ${DB_CONNECTION_POOL_SIZE_CLINEPI}
-
       DB_CONNECTION_ENABLED_MICROBIOME: ${DB_CONNECTION_ENABLED_MICROBIOME}
       DB_CONNECTION_NAME_MICROBIOME: ${DB_CONNECTION_NAME_MICROBIOME:?}
-      DB_CONNECTION_LDAP_MICROBIOME: ${DB_CONNECTION_LDAP_MICROBIOME:?}
+      DB_CONNECTION_LDAP_MICROBIOME: ${DB_CONNECTION_LDAP_MICROBIOME}
```

Comment:
Oh, does ? make it required?  I was thinking absence of a default made it choke if no value was supplied.

Reply:
Yeah, I think it defaults to an empty string if it's unspecified.